### DISCUSSION TITLE
Dev set-up: non-std docker postgres port?

### DISCUSSION THREAD

**Discussion by weedySeaDragon** [freelawproject/bigcases2#272] [Q&A] [answered]
@ERosendo - 
I just set up a development environment so I can work on #230. 
I ran into a bump when setting up and running the docker containers:

The host port for the `bc2-postgresql` container is set to `5433` instead of the postgresql standard `5432`.  _Is this as it should be?_  If so, why?

Here's the section in the docker-compose file:

`docker/bigcasesbot/docker-compose.yml`
```docker
  bc2-postgresql:
    container_name: bc2-postgresql
    ports:
      - "5433:5432"
```

If this _should_ be corrected to `- "5432:5432"` I'll submit a PR

**Comment by mlissner** [comment, marked as answer]
This is on purpose so folks can work on CourtListener and bots at the same time.
